monitor.cpp

```diff
@@ -1,11 +1,48 @@
 #include "./monitor.h"
+#include <stdlib.h>
+#include <stdio.h>
 #include <assert.h>
+#include <locale>
+#include <codecvt>
 #include <thread>
 #include <chrono>
 #include <iostream>
+#include <vector>
+#include <map>
+#include <string>
+
 using std::cout, std::flush, std::this_thread::sleep_for, std::chrono::seconds;
+enum Language {
+    TAMIL,
+    GREEK
+//  DUTCH supported in future
+};
+std::vector<std::wstring> Tamil;
+std::vector<std::wstring> Greek;
+std::map<Language, std::vector<std::wstring>> warningMessages;
```

Comment:
Nice data-structure for messages!
However, the strings aren't automatically tested. Also, it's hard to test all the situations in all the languages manually each time. Of course, the "understand-ability" of the translation can only be confirmed manually by native speakers. But other things can go wrong, like missing a translation in a particular language, etc.

Can you think of any asserts to safeguard against that?

Reply:
introduced the testcase for testing the displayWarningMessages.
Thanks for commenting for it.